double/double.mbti

```diff
@@ -19,21 +19,29 @@ let not_a_number : Double
 impl Double {
   abs(Double) -> Double
   acos(Double) -> Double
+  acosh(Double) -> Double
   asin(Double) -> Double
+  asinh(Double) -> Double
   atan(Double) -> Double
   atan2(Double, Double) -> Double
+  atanh(Double) -> Double
+  cbrt(Double) -> Double
   ceil(Double) -> Double
   cos(Double) -> Double
+  cosh(Double) -> Double
   exp(Double) -> Double
+  expm1(Double) -> Double
   floor(Double) -> Double
   from_int(Int) -> Double
+  hypot(Double, Double) -> Double
   inf(Int) -> Double //deprecated
   is_close(Double, Double, relative_tolerance~ : Double = .., absolute_tolerance~ : Double = ..) -> Bool
   is_inf(Double) -> Bool
   is_nan(Double) -> Bool
   is_neg_inf(Double) -> Bool
   is_pos_inf(Double) -> Bool
   ln(Double) -> Double
+  ln_1p(Double) -> Double
```

Comment:
ln1p looks more consistent with log10 and log2 to me

Reply:
I think adding aliases will make reading MoonBit code more difficult - I might think `log1p` and `ln_1p` refers to two different functions if I were the reader.